circus-train-tool-parent/circus-train-tool-core/src/test/java/com/hotels/bdp/circustrain/tool/core/endpoint/ReplicaHiveEndpointTest.java

```diff
@@ -29,7 +29,7 @@
 
 import com.google.common.base.Supplier;
 
-import com.hotels.bdp.circustrain.api.metastore.CloseableMetaStoreClient;
+import com.hotels.hcommon.hive.metastore.client.api.CloseableMetaStoreClient;
```

Comment:
hcommon after bdp... (strange as most of your other files have this in the correct order...)

Reply:
ping ^